packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -25,7 +25,8 @@ import Services from './services';
 import LoggerProxy from './logger-proxy';
 import {StateChange, Logout} from './services/agent/types';
 import {ConnectionService} from './services/core/WebSocket/connection-service';
-import {getErrorDetails} from './services/core/Utils';
+import {getErrorDetails, getErrorReason} from './services/core/Utils';
+import {EventBus} from './services/core/EventBus';
```

Comment:
Have added an `EventBus` in order to have an easy way to manage events. Will explain this in further detail.